unlock-app/src/stories/lock/ConfirmingKeyLock.stories.js

```diff
@@ -43,6 +45,7 @@ storiesOf('ConfirmingKeyLock', ConfirmingKeyLock)
       />
     )
   })
+/* obsolte, preserving in case design changes later
```

Comment:
You can probably remove that.. let's keep things clean, it's easy enough to add again when/if we need it again in the future.